include/picongpu/particles/atomicPhysics/AtomicPhysics.kernel

```diff
@@ -0,0 +1,362 @@
+                        << uint16_t(levelVector[5]) << ", "
+                        << uint16_t(levelVector[6]) << ", "
+                        << uint16_t(levelVector[7]) << ", "
+                        << uint16_t(levelVector[8]) << ", "
+                        << uint16_t(levelVector[9]) << ", "
+                        << std::endl;
+
+                    ConfigNumberDataType newState = 352973u;
+                    auto levelVector2 = ConfigNumber::getLevelVector(newState);
+                    std::cout << "newState: { " << uint16_t(levelVector2[0]) << ", "
+                        << uint16_t(levelVector2[1]) << ", "
+                        << uint16_t(levelVector2[2]) << ", "
+                        << uint16_t(levelVector2[3]) << ", "
+                        << uint16_t(levelVector2[4]) << ", "
+                        << uint16_t(levelVector2[5]) << ", "
+                        << uint16_t(levelVector2[6]) << ", "
+                        << uint16_t(levelVector2[7]) << ", "
+                        << uint16_t(levelVector2[8]) << ", "
+                        << uint16_t(levelVector2[9]) << ", "
+                        << std::endl;
+
+                    std::cout << "oldStateenergy " << float_64(atomicDataBox(oldState)) * UNITCONV_AU_to_eV <<
+                    std::endl; std::cout << "newStateenergy " << float_64(atomicDataBox(newState)) * UNITCONV_AU_to_eV
+                    << std::endl;
+
+                    std::cout << "energyDifference "
+                        << float_64(AtomicRate::energyDifference(acc, oldState, newState, atomicDataBox)) *
+                    UNITCONV_AU_to_eV << std::endl;
+
+                    auto transitionIndex = atomicDataBox.findTransitionInBlock(atomicDataBox.findState(oldState),
+                    newState); std::cout << "transitionIndex "
+                        << transitionIndex << std::endl;
+
+                    std::cout << "gauntFactor " << AtomicRate::gauntFactor(
+                        AtomicRate::energyDifference(acc, oldState, newState, atomicDataBox),
+                        5._X,
+                        transitionIndex,
+                        atomicDataBox) << std::endl;
+
+                    std::cout << "cross section " << AtomicRate::collisionalExcitationCrosssection(
+                        acc,
+                        oldState,
+                        newState,
+                        transitionIndex,
+                        5._X,
+                        atomicDataBox) << std::endl;
+
+                    std::cout << "Rate " << AtomicRate::Rate(
+                        acc,
+                        oldState,
+                        newState,
+                        transitionIndex,
+                        5._X,
+                        10._X,
+                        float_X(9.99999e24),
+                        atomicDataBox) << std::endl;
+                    */
+
+                    // debug only
+                    if(workerIdx == 0)
```

Comment:
I know that this code is still experimental but we should not keep this print because each block will spam to the terminal.

Reply:
done